packages/sitecore-jss-proxy/src/middleware/editing/index.ts

```diff
@@ -0,0 +1,119 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import { debug } from '@sitecore-jss/sitecore-jss';
+import {
+  EDITING_ALLOWED_ORIGINS,
+  QUERY_PARAM_EDITING_SECRET,
+} from '@sitecore-jss/sitecore-jss/editing';
+import { enforceCors } from '@sitecore-jss/sitecore-jss/utils';
+import { EditingConfigEndpointOptions, editingConfigMiddleware } from './config';
+
+/**
+ * Default endpoints for editing requests
+ */
+const ENDPOINTS = {
+  CONFIG: '/config',
+  RENDER: '/render',
+};
+
+/**
+ * Configuration for the editing router
+ */
+export type EditingRouterConfig = {
+  /**
+   * Configuration for the /config endpoint
+   */
+  config: EditingConfigEndpointOptions;
+  /**
+   * Configuration for the /render endpoint
+   */
+  render?: {
+    /**
+     * Custom path for the editing render endpoint
+     */
+    path?: string;
+  };
+};
+
+/**
+ * Middleware to handle editing requests
+ * @param {Request} req Request
+ * @param {Response} res Response
+ * @param {NextFunction} next Next function
+ */
+export const editingMiddleware = async (
```

Comment:
@art-alexeyenko It's a good point, I thought about that as well
In order to do that it requires to refactor the next.js middlewares (editing, render, feaas-render) - it's a bit bigger and not an easy replacement.
Regarding future frameworks - we will introduce a framework-agnostic middleware in JSS V2
Just in case if we will introduce support for new frameworks in V1 (Meta frameworks), we can do that. I think the "/editing/config" middleware can be reused as well across the frameworks, not really sure about "/editing/render" for now